eu/intermodalics/tango_nodelet_manager/maven-metadata.xml

```diff
@@ -3,12 +3,10 @@
   <groupId>eu.intermodalics</groupId>
   <artifactId>tango_nodelet_manager</artifactId>
   <versioning>
-    <release>1.2.0</release>
+    <release>1.3.0</release>
     <versions>
-      <version>1.1.0</version>
-      <version>1.1.1</version>
-      <version>1.2.0</version>
```

Comment:
This one looks a bit weird to me. This file generally keeps the old versions as well, and writes the latest one inside the `release` tag. I don't remember exactly how to do that, but I think that's the way it should be (at least it was done like that until now).